installer/image_push/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Remove local images to ensure proper push behavior
+  block:
+    - name: Remove web image
+      docker_image:
+        name: "{{ docker_registry }}/{{ docker_registry_repository }}/{{ awx_web_image }}"
+        tag: "{{ awx_version }}"
+        state: absent
+
+    - name: Remove task image
+      docker_image:
+        name: "{{ docker_registry }}/{{ docker_registry_repository }}/{{ awx_task_image }}"
+        tag: "{{ awx_version }}"
+        state: absent
+  delegate_to: localhost
+  when: docker_remove_local_images
```

Comment:
When using this do a `default(False)` that way we can comment out the line in the inventory file. This is a general direction I've been trying to head in.

Reply:
I'm getting a warning to not use jinja2 templating delimiters in a when statement.

What's the recommended way to approach this?

Reply:
I want to target ansible 2.4+ so if it's giving a warning then we should move to the latest recommended approach

Reply:
Here's my current setup:

```
ansible 2.4.0.0
  config file = None
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.12 (default, Sep  1 2016, 22:14:00) [GCC 4.8.3 20140911 (Red Hat 4.8.3-9)]
```

Here's the exact warning:
```
 [WARNING]: when statements should not include jinja2 templating delimiters such as {{ }} or {% %}. Found: {{ docker_remove_local_images |
default(False) }}
```

Do you know what the latest recommended approach is?  I did some digging and it wasn't immediately obvious what the preferred way to this is.  Leaving the variable uninitialized is an option, but it requires the somewhat unwieldy `when: docker_remove_local_images is defined and docker_remove_local_images` and just generally feels like a bad practice to me.

Reply:
I think you can just omit the jinja template wrappers right?

So instead of `when: {{ var|default(False) }}`
you can say: `when: var|default(False)`

Reply:
Oh, cool!  Learned something new.